dotnet-desktop-guide/net/winforms/controls/how-to-set-the-tab-order.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Set tab order of controls
+description: Learn how to set the tab order of controls on your Windows Forms for .NET. Set the tab order with Visual Studio or using the TabIndex property in the Properties window.
+ms.date: 05/24/2021
+dev_langs:
+  - "csharp"
+  - "vb"
+helpviewer_keywords:
+  - "tab order [Windows Forms], controls on Windows forms"
+  - "Windows Forms controls, setting tab order"
+  - "controls [Windows Forms], setting tab order"
+  - "Windows Forms, setting tab order"
+---
+# How to set the tab order on Windows Forms (Windows Forms .NET)
+
+The tab order is the order in which a user moves focus from one control to another by pressing the <kbd>Tab</kbd> key. Each form has its own tab order. By default, the tab order is the same as the order in which you created the controls. Tab-order numbering begins with zero and ascends in value, and is set with the <xref:System.Windows.Forms.Control.TabIndex%2A> property.
+
+You can also set the text by using the [designer](#designer).
+
+[!INCLUDE [desktop guide under construction](../../includes/desktop-guide-preview-note.md)]
+
+Tab order can be set in the **Properties** window of the designer using the <xref:System.Windows.Forms.Control.TabIndex%2A> property. The `TabIndex` property of a control determines where it's positioned in the tab order. By default, the first control added to the designer has a `TabIndex` value of 0, the second has a `TabIndex` of 1, and so on. Once the highest `TabIndex` has been focused, pressing <kbd>Tab</kbd> will cycle and focus the control with the lowest `TabIndex` value.
+
+Container controls, such as a <xref:System.Windows.Forms.GroupBox> control, treat its children separate from the rest of the form. Each child in the container has its own <xref:System.Windows.Forms.Control.TabIndex%2A> value. Because a container control can't be focused, when the tab order reaches the container control, the first child control of the container with the lowest `TabIndex` is focused. As the <kbd>Tab</kbd> is pressed, each of the child is focused according to its `TabIndex` value until the last control. When <kbd>Tab</kbd> is pressed on the last control, focus resumes to the next control in the parent of the container, based on the next `TabIndex` value.
+
+Any control of the many on your form can be skipped in the tab order. Usually, pressing <kbd>Tab</kbd> successively at run time selects each control in the tab order. By turning off the <xref:System.Windows.Forms.Control.TabStop%2A> property, a control is passed over in the tab order of the form.
+
+## Designer
+
+The Visual Studio designer provides two ways to set the tab order of a control.
```

Comment:
This promises two ways but is followed by only one way.

Reply:
```suggestion
Use the Visual Studio designer **Properties** window to set the tab order of a control.
```